### PR TITLE
scaleway: Misc doc changes

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

* Updated example section for ``variables``
* Added link about token generation
* Misc changes in doc

Fixes: #467

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/inventory/scaleway.py

### CODE REVIEW

**Comment by Akasurde** [comment]
This is a minor doc change so didn't add changelog fragment.

File: plugins/inventory/scaleway.py [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
         tags:
-            description: Filter results on a specific tag
+            description: Filter results on a specific tag.
             type: list
         oauth_token:
             required: True
-            description: Scaleway OAuth token.
+            description:
+            - Scaleway OAuth token.
+            - More details on how to generate L(token, https://www.scaleway.com/en/docs/generate-api-keys/).
```
Comment (felixfontein) [inline plugins/inventory/scaleway.py:35]: That way it's easier to use with a screen reader.
```suggestion
            - More details on L(how to generate token, https://www.scaleway.com/en/docs/generate-api-keys/).
```

File: plugins/inventory/scaleway.py [hunk 2]
```diff
@@ -1,36 +1,38 @@
...
     name: scaleway
     author:
       - Remy Leone (@sieben)
     short_description: Scaleway inventory source
     description:
-        - Get inventory hosts from Scaleway
+        - Get inventory hosts from Scaleway.
     options:
         plugin:
             description: token that ensures this is a source file for the 'scaleway' plugin.
```
Comment (Andersson007) [inline plugins/inventory/scaleway.py:17]:
```suggestion
            description: Token that ensures this is a source file for the 'scaleway' plugin.
```
There's also the `variables` option's description to capitalize the first letter.

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@Akasurde thanks for the PR! @felixfontein thanks for reviewing!

**Comment by Akasurde** [comment]
@Andersson007 @felixfontein Thanks for the review and merge.

**Pull request merged**